commands.md

```diff
@@ -17,6 +17,8 @@ Command | Implemented By | Who can run it | Description
 `/reopen` | prow [lifecycle](./prow/plugins/lifecycle) | authors and assignees | reopens a closed issue/PR
 `/lifecycle [state]` | prow [lifecycle](./prow/plugins/lifecycle) | anyone | adds a stale, rotten or frozen state label
 `/remove-lifecycle [state]` | prow [lifecycle](./prow/plugins/lifecycle) | anyone | removes a stale, rotten or frozen state label
+`/help` | prow [help](./prow/plugins/help) | anyone | adds the `help wanted` label
+`/remove-help` | prow [help](./prow/plugins/help) | anyone | removes the `help wanted` label
```

Comment:
The `remove-` prefix is used with commands that accept a target parameter to specify the label to remove. I think `/help cancel` might be more consistent with how other commands that toggle a single label (`/lgtm` and `/hold`) are used.

I wish our commands had a more standard format. We have `un` and `remove-` prefixes in addition to ` cancel` parameters :-1:

Reply:
I think we should standardize on `/remove-foo`